doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -690,10 +695,14 @@ side. E.g.:
 Summary
 ~~~~~~~
 
-**Syntax of notations**
+.. _NotationSyntax:
+
+Syntax of notations
++++++++++++++++++++
 
 The different syntactic variants of the command Notation are given on the
-following figure. The optional :token:`scope` is described in the Section 12.2.
+following figure. The optional :token:`scope` is described in
+:ref:`Scopes`.
```

Comment:
As you can see the token `scope` is not clickable because it wasn't defined anywhere. I don't really know whether it should be added to the grammar below, or be defined in the text by replacing the use of ```:token:`scope` ``` at one point by ```:production:`scope` ``` as described here https://github.com/coq/coq/tree/master/doc/sphinx#coq-roles. By looking at the Coq parser, we can see that scopes are just idents. So I guess it makes sense to not extend the grammar (like types which are just special kinds of terms, scopes are just idents with a special role) but to use an inline `production` instead.

Reply:
Okay, let's use `production`.